content/working_with/official_plugins/Infrastructure/aws.md

```diff
@@ -7035,6 +7035,26 @@ For more information, and possible keyword arguments, see: [SQS Queue:create_que
             MessageRetentionPeriod: '86400'
             VisibilityTimeout: '180'
 ```
+## **Known Issues**
```

Comment:
This explanation should also be added to the Q&A page in the examples section

content/trial_getting_started/trial_faq